client/my-sites/hosting/staging-site-card/card-content/manage-staging-site-card-content.tsx

```diff
@@ -104,6 +106,24 @@ export const ManageStagingSiteCardContent = ( {
 			);
 		};
 
+		const ConfirmationPullChangesButton = () => {
+			return (
+				<ConfirmationModal
+					disabled={ isButtonDisabled }
+					onConfirm={ onPullClick }
+					modalTitle={ translate( 'Confirm pull your changes to your production site' ) }
```

Comment:
```suggestion
					modalTitle={ translate( 'Confirm pull your changes from your staging site' ) }
```

We will adjust it when we apply the design, but let's clarify those to use the same wording as in the button.